src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -33,15 +34,28 @@ export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
         placeholder: 'Enter your reason',
         value: reason,
         onChange: setReason
-      })
+      }),
+      div([
+        h(Checkbox, {
+          id: 'send-copy_link',
+          checked: sendCopy,
+          onChange: () => setSendCopy(!sendCopy)
+        }),
+        label({ htmlFor: 'send-copy_link', style: { marginLeft: 5 } }, ['Also send me a copy'])
+      ])
     ])]),
     table({ style: { margin: '1rem', width: '100%' } }, [
       thead([
         tr({ style: { height: '2rem' } }, [th({ style: { textAlign: 'left' } }, ['Datasets']), th({ style: { textAlign: 'left', width: '15rem' } }, ['Access'])])
       ]),
       tbody(
-        _.map(({ 'dct:title': title, access, id }) => tr({ key: id, style: { height: '2rem' } }, [
-          td({ style: { paddingRight: 20 } }, [title]),
+        _.map(({ 'dct:title': title, access, id, contacts }) => tr({ key: id, style: { height: '2rem' } }, [
+          td({ style: { paddingRight: 20 } }, [
+            title,
+            div({ style: { fontSize: '.7rem', marginTop: 5, width: 'fit-content' } }, [
+              _.map(({ email }) => email && h(Link, { key: `${id}_${email}`, href: `mailto:${email}`, style: { marginTop: 5, display: 'block' } }, [email]), contacts)
```

Comment:
Since keys only have to be unique amongst their peers, I think this one can just be `email`